userguide/virtualmachines.rst

```diff
@@ -68,44 +68,41 @@ wizard:
 
 #. **Select VM wizard type**
 
-   Select :literal:`Virtual Macine (VM)` or
-   :literal:`Docker Host`.
+   Select *Virtual Macine (VM)* or
+   *Docker Host*.
 
 #. **Operating System**
 
    Select the guest operating system type from the drop down menu and
-   enter an alphanumeric name for the VM. It is recommended to use UEFI
-   as the boot method for newer operating systems. The settings
+   enter an alphanumeric name for the VM. UEFI is recommended
+   as the boot method for newer operating systems.
    :guilabel:`Start on Boot` and :guilabel:`Enable VNC` are set by default.
-   The setting :guilabel:`Start on Boot` tells the VM to start when the
-   host system is booted. The setting :guilabel:`Enable VNC` allows a
+   :guilabel:`Start on Boot` tells the VM to start when the
+   host system is booted. :guilabel:`Enable VNC` allows a
    virtual connection to the VM.
    
 #. **CPU and Memory**
 
-   Enter the amount of virtual CPUs for the VM to use. Enter the amount
-   of memory, in MiB,for the VM to use. Not using adequate memory or
-   CPUs can affect the performance of the vm. Using too much memory or
-   CUPs can affect the performance of the host system.
+   Enter the number of virtual CPUs for the VM to use. Enter the amount
```

Comment:
While short sentences are generally better, this seems a bit choppy.  Maybe combine the first two and the last two?
"Enter the number of virtual CPUs and the amount of memory for the VM. Provide adequate memory and virtual CPUs for the applications running in the VM, but be aware that providing more memory or virtual CPUs than are needed can affect performance of the host system."